README.md

```diff
@@ -91,17 +88,57 @@ module.exports = {
   networks: {
     development: {
       host: "localhost",
-      port: 8545,
+      port: 8551,
       network_id: "*" // Match any network id
     },
-    testnet: {
-      provider: () => new HDWalletProvider(mnemonic, "https://api.baobab.klaytn.net:8651"),
+    klaytn: {
+      provider: () => {
+        const pks = JSON.parse(fs.readFileSync(path.resolve(__dirname)+'/privateKeys.js'))
+
+        return new HDWalletProvider(pks, "http://localhost:8551", 0, pks.length)
```

Comment:
Why does the length of the key needed here?

Reply:
privateKeys.js contains multiple private keys. pks.length = number of private keys in the file.

Reply:
Does it have a format like ?: 
```
["PK1",
"PK2",
"PK3"]
```

Reply:
Yes.